examples/flask/hello.py

```diff
@@ -82,6 +91,35 @@ def show_post(post_id):
 def get_tasks():
     return jsonify({'tasks': tasks})
 
+response = [
+    {
+        'id': 1,
+        'title': u'Buy groceries',
+        'description': u'Milk, Cheese, Pizza, Fruit, Tylenol',
+        'done': False
+    },
+    {
+        'id': 2,
+        'title': u'Learn Python',
+        'description': u'Need to find a good Python tutorial on the web',
+        'done': False
+    }
+]
+
+@app.route('/test/html_response', methods=['GET'])
+def html_response():
+    return """
+    <p>Hello</p>
+    <p>world</p>
+    """
+
+@app.route('/test/xml_response', methods=['GET'])
+def xml_response():
+    return Response(response='TEST XML OK', status = 200, mimetype='application/xml')
```

Comment:
could we test and update the example with valid xml body?

Reply:
tested